src/cryptography/hazmat/bindings/openssl/ssl.py

```diff
@@ -7,6 +7,14 @@
 INCLUDES = """
 #include <openssl/ssl.h>
 
+/* LibreSSL has removed support for compression, and with it the
+ * COMP_METHOD use in ssl.h. This is a hack to make the function types
+ * in this code match those in ssl.h.
+ */
+#ifdef LIBRESSL_VERSION_NUMBER
+#define COMP_METHOD void
+#endif
```

Comment:
Sorry, move what?

On Thu, Feb 19, 2015 at 2:53 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/bindings/openssl/ssl.py
> <https://github.com/pyca/cryptography/pull/1679#discussion_r25019164>:
>
> > @@ -7,6 +7,14 @@
> >  INCLUDES = """
> >  #include <openssl/ssl.h>
> >
> > +/* LibreSSL has removed support for compression, and with it the
> > + * COMP_METHOD use in ssl.h. This is a hack to make the function types
> > + * in this code match those in ssl.h.
> > + */
> > +#ifdef LIBRESSL_VERSION_NUMBER
> > +#define COMP_METHOD void
> > +#endif
>
> Yeah, although that starts to bleed into #1675
> <https://github.com/pyca/cryptography/pull/1675> [image: :grinning:]
>
> I'll move this so we can merge today.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1679/files#r25019164>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084